bin/cylc-jobscript

```diff
@@ -34,26 +34,56 @@ Other options (e.g. for suite host and owner) are passed
 through to the submit command.
 
 Options:
-  -h,--help   - print this usage message.
+  -h, --help   Print this usage message.
+  -e --edit    Open the jobscript in a CLI text editor.
+  -g --gedit   Open the jobscript in a GUI text editor.
+  --plain      Don't print the "Task Job Script Generated message."
  (see also 'cylc submit --help')
 
 Arguments:
-  REG         - Registered suite name.
-  TASK        - Task ID (NAME.CYCLE_POINT)
+  REG          Registered suite name.
+  TASK         Task ID (NAME.CYCLE_POINT)
 __END__
 }
 
+editor () {
+    python -c "
+import sys
+from cylc.cfgspec.globalcfg import GLOBAL_CFG
+if sys.argv[1] in ['-g', '--gedit']:
+    print GLOBAL_CFG.get(['editors', 'gui'])
+elif sys.argv[1] in ['-e', '--edit']:
+    print GLOBAL_CFG.get(['editors', 'terminal'])
+" $1
+}
+
+PLAIN=0
```

Comment:
With boolean, worth doing `PLAIN=false` and `PLAIN=true`, so you can simply do `if "${PLAIN}"; then` later on.